meetings/StackStorm_TSC_Meetings_009l2r7sjp9ojnni0v22jjcrfg@group.calendar.google.com.ics

```diff
@@ -0,0 +1,53 @@
+BEGIN:VCALENDAR
+PRODID:-//Google Inc//Google Calendar 70.9054//EN
+VERSION:2.0
+CALSCALE:GREGORIAN
+METHOD:PUBLISH
+X-WR-CALNAME:StackStorm TSC Meetings
+X-WR-TIMEZONE:America/New_York
+BEGIN:VTIMEZONE
+TZID:America/Los_Angeles
+X-LIC-LOCATION:America/Los_Angeles
+BEGIN:DAYLIGHT
+TZOFFSETFROM:-0800
+TZOFFSETTO:-0700
+TZNAME:PDT
+DTSTART:19700308T020000
+RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=2SU
+END:DAYLIGHT
+BEGIN:STANDARD
+TZOFFSETFROM:-0700
+TZOFFSETTO:-0800
+TZNAME:PST
+DTSTART:19701101T020000
+RRULE:FREQ=YEARLY;BYMONTH=11;BYDAY=1SU
+END:STANDARD
+END:VTIMEZONE
+BEGIN:VEVENT
+DTSTART;TZID=America/Los_Angeles:20200407T093000
+DTEND;TZID=America/Los_Angeles:20200407T103000
+RRULE:FREQ=MONTHLY;BYDAY=1TU
+DTSTAMP:20200826T143010Z
+UID:2f4s2ik23m09tnlb7p9as01lt8@google.com
+CREATED:20200826T142640Z
+DESCRIPTION:<p><span></span></p><br>StackStorm Technical Steering Committee

```

Comment:
Would be also nice to include the discussion link to TSC meeting with more details: https://github.com/StackStorm/discussions/issues/33